stock_move_line_reserved_quant/README.rst

```diff
@@ -62,6 +62,7 @@ Contributors
 ------------
 
 -  Denis Roussel <denis.roussel@acsone.eu>
+-  SodexisTeam <dev@sodexis.com>
```

Comment:
@OCA/core-maintainers I'm still concerned by contributors as a team (anonymous)

Reply:
@rousseldenis I too prefer to see people's names, but I personally don't see it as a problem.